_docs/deployment/logs.md

```diff
@@ -76,15 +76,47 @@ If your application logs are output in JSON, they will be easily searchable in [
 
 ## How to automatically copy your logs elsewhere
 
-If you want to set up your own storage for your application logs, you can set up [a "log drain" service](https://docs.cloudfoundry.org/devguide/services/log-management.html) that sends the logs to your preferred location.
+If you want to set up your own storage for your application logs, you can set up [a "log drain" service](https://docs.cloudfoundry.org/devguide/services/log-management.html#user-provided) that sends the logs to your preferred location.
 
-Create the user provided service and point it toward the endpoint where you want to send your logs: 
+Create the user provided service and point it toward the endpoint where you want to send your logs:
+
+      cf create-user-provided-service my-log-drain -l syslog-tls://<your-log-drain-service-endpont>
```

Comment:
```suggestion
      cf create-user-provided-service my-log-drain \
        -l syslog-tls://<your-log-drain-service-endpont>
```
![Screenshot 2024-09-05 at 3 15 36 PM](https://github.com/user-attachments/assets/108fa184-2845-4777-b402-ac9d0185d488)

Breaking command into multiple lines, otherwise the command text extends past the edge of the formatted code box